sdk/identity/identity/src/credentials/managedIdentityCredential/index.ts

```diff
@@ -84,7 +85,7 @@ export class ManagedIdentityCredential implements TokenCredential {
       }
     }
 
-    throw new CredentialUnavailableError("ManagedIdentityCredential - No MSI credential available");
+    throw new CredentialUnavailableError(`${credentialName} - No MSI credential available`);
```

Comment:
Rather than redefining the class name as a const variable, you can use `this.constructor.name` instead or `ManagedIdentityCredential.name` if you want to avoid getting the name of a class that extends it instead.